js-arrays-objects/js-objects-and-arrays_objects/index.js

```diff
@@ -15,8 +18,9 @@ const personAlex = {
   age: 24,
 };
 
-const nameOfAlex = "Change me";
-const ageOfAlex = "Change me";
+const nameOfAlex = personAlex.name;
```

Comment:
I think this wants you to change the values in your object e.g:

`const personWithNameAgeEmail = {
   name: personAlex.name,
   age: personAlex.age,
   email: "xxz@gmail.com",
 };`